platform/ios/Integration Tests/MBGLIntegrationTests.m

```diff
@@ -1,76 +1,17 @@
-#import <XCTest/XCTest.h>
+#import "MGLMapViewIntegrationTest.h"
 #import <objc/message.h>
```

Comment:
I don’t see any `objc_msgSend`, so maybe we can remove this import?

Reply:
Good catch - that's a left-over